client/client-multi/client-java-core/src/main/java/org/kaaproject/kaa/client/Kaa.java

```diff
@@ -16,21 +16,20 @@
 
 package org.kaaproject.kaa.client;
 
-import java.io.IOException;
-import java.security.GeneralSecurityException;
-
 import org.kaaproject.kaa.client.exceptions.KaaInvalidConfigurationException;
 import org.kaaproject.kaa.client.exceptions.KaaRuntimeException;
 import org.kaaproject.kaa.client.exceptions.KaaUnsupportedPlatformException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+
 /**
  * Creates new Kaa client based on {@link KaaClientPlatformContext platform
  * context} and optional {@link KaaClientStateListener state listener}.
- * 
```

Comment:
Missing description of <b>isUserKeyStrategy</b> parameter, and why it's named <b>isUserKeyStrategy</b> ?